### PR TITLE
plr: init at 8.4.1

### PR DESCRIPTION
**Pull request by qoelet**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by doronbehar — CHANGES_REQUESTED** [review]
You forgot to add a line in all-packages.nix.

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+stdenv.mkDerivation rec {
+  name = "plr";
+  src = fetchFromGitHub {
+    owner = "postgres-plr";
+    repo = "plr";
+    rev = "c82fe1984b51f53ce7a4ad6853815bc9554fa390";
+    sha256 = "1cnjkrxk9s14n7p25bv7zwibw8q379ajhqpik66rw80mwci44xka";
+  };
+
+  buildInputs = [ R postgresql pkgconfig ];
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:12]:
```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [ R postgresql ];
```

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pkgconfig, R, postgresql }:
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:1]:
```suggestion
{ stdenv, fetchFromGitHub, pkg-config, R, postgresql }:
```

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    install -D plr.so -t $out/lib/
+    install -D {plr--unpackaged--8.4.sql,plr--8.4.sql,plr--8.3.0.18--8.4.sql,plr.control} -t $out/share/postgresql/extension
+  '';
+
+  meta = with stdenv.lib; {
+    description = "PL/R - R Procedural Language for PostgreSQL";
+    homepage = "https://github.com/postgres-plr/plr";
+    maintainers = with maintainers; [ qoelet ];
+    platforms = [ "x86_64-linux" ];
+    license = licenses.gpl2;
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:24]: Plus or Only?

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  };
+
+  buildInputs = [ R postgresql pkgconfig ];
+  buildPhase = "USE_PGXS=1 make";
+  installPhase = ''
+    install -D plr.so -t $out/lib/
+    install -D {plr--unpackaged--8.4.sql,plr--8.4.sql,plr--8.3.0.18--8.4.sql,plr.control} -t $out/share/postgresql/extension
+  '';
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:19]:
```suggestion
  meta = with lib; {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/servers/sql/postgresql/ext/plr.nix (hunk 3 above)
Comment (qoelet) [reply pkgs/servers/sql/postgresql/ext/plr.nix]: Amended to Only, thanks.

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkg-config, R, postgresql }:
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:1]:
```suggestion
{ lib, stdenv, fetchFromGitHub, pkg-config, R, postgresql }:
```

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchFromGitHub {
+    owner = "postgres-plr";
+    repo = "plr";
+    rev = "c82fe1984b51f53ce7a4ad6853815bc9554fa390";
+    sha256 = "1cnjkrxk9s14n7p25bv7zwibw8q379ajhqpik66rw80mwci44xka";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ R postgresql ];
+  buildPhase = "USE_PGXS=1 make";
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:14]:
```suggestion

  preBuild = ''
    export USE_PGXS=1
  '';

```

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 7]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkg-config, R, postgresql }:
+
+stdenv.mkDerivation rec {
+  name = "plr";
+  src = fetchFromGitHub {
+    owner = "postgres-plr";
+    repo = "plr";
+    rev = "c82fe1984b51f53ce7a4ad6853815bc9554fa390";
+    sha256 = "1cnjkrxk9s14n7p25bv7zwibw8q379ajhqpik66rw80mwci44xka";
+  };
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:10]:
```suggestion
  pname = "plr";
  version = "8.4.1";

  src = fetchFromGitHub {
    owner = "postgres-plr";
    repo = "plr";
    rev = "REL${builtins.replaceStrings ["."] [":"] version}";
    sha256 = "1cnjkrxk9s14n7p25bv7zwibw8q379ajhqpik66rw80mwci44xka";
  };
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please change the title and commit version number to 8.4 or 8.4.1.

File: pkgs/servers/sql/postgresql/ext/plr.nix (hunk 7 above)
Comment (qoelet) [reply pkgs/servers/sql/postgresql/ext/plr.nix]: @SuperSandro2000 updated the PR, now set to 8.4.1.

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    sha256 = "1wy4blg8jl30kzhrkvbncl4gmy6k71zipnq89ykwi1vmx89v3ab7";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ R postgresql ];
+  preBuild = ''
+    export USE_PGXS=1
+  '';
+  installPhase = ''
+    echo $out
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:20]:
```suggestion
  '';

  installPhase = ''
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 91155` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>postgresqlPackages.plr (postgresql11Packages.plr)</li>
</ul>
</details>

File: pkgs/servers/sql/postgresql/ext/plr.nix [hunk 9]
```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, R, postgresql }:
+
+stdenv.mkDerivation rec {
+  name = "plr";
```
Comment (SuperSandro2000) [inline pkgs/servers/sql/postgresql/ext/plr.nix:4]:
```suggestion
  pname = "plr";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**